streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -755,9 +755,9 @@ private void runLoop() {
             try {
                 recordsProcessedBeforeCommit = runOnce(recordsProcessedBeforeCommit);
             } catch (final TaskMigratedException ignoreAndRejoinGroup) {
-                log.warn("Detected a task that got migrated to another thread. " +
+                log.warn("Detected {} that got migrated to another thread. " +
                     "This implies that this thread missed a rebalance and dropped out of the consumer group. " +
-                    "Trying to rejoin the consumer group now.", ignoreAndRejoinGroup);
+                    "Trying to rejoin the consumer group now:\n{}", ignoreAndRejoinGroup.migratedTask().id(), ignoreAndRejoinGroup.migratedTask());
```

Comment:
What does the final "\n{}" do? It prints out the migratedTask (using toString() I assume)? What does a task.toString() look like? 

In other words, I'm trying to understand what this log line will look like.

Reply:
It will contain the following info (I used an example from production logs but removed sensitive names):

```
WARN [thread-name] Detected task 1_0 that got migrated to another thread. This implies that this thread missed a rebalance and dropped out of the consumer group. Trying to rejoin the consumer group now.

> StreamsTask taskId: 1_0
>	ProcessorTopology:
>		KSTREAM-SOURCE-0000000003:
			topics:		[tec1.trex-jcheng.trex-delta.replicatorFullExportPartitioned]
			children:	[KSTREAM-TRANSFORM-0000000004]
		KSTREAM-TRANSFORM-0000000004:
			states:		[ThingsToDeleteAtTheEnd, CurrentValues]
			children:	[KSTREAM-BRANCH-0000000005]
		KSTREAM-BRANCH-0000000005:
			children:	[KSTREAM-BRANCHCHILD-0000000006, KSTREAM-BRANCHCHILD-0000000007, KSTREAM-BRANCHCHILD-0000000008]
		KSTREAM-BRANCHCHILD-0000000006:
			children:	[KSTREAM-MAP-0000000009]
		KSTREAM-MAP-0000000009:
			children:	[KSTREAM-SINK-0000000010]
		KSTREAM-SINK-0000000010:
			topic:		tec1.trex-jcheng.trex-delta.pgd-deltas-showings
		KSTREAM-BRANCHCHILD-0000000007:
			children:	[KSTREAM-MAP-0000000011]
		KSTREAM-MAP-0000000011:
			children:	[KSTREAM-SINK-0000000012]
		KSTREAM-SINK-0000000012:
			topic:		tec1.trex-jcheng.trex-delta.pgd-deltas-program
		KSTREAM-BRANCHCHILD-0000000008:
			children:	[KSTREAM-MAP-0000000013]
		KSTREAM-MAP-0000000013:
			children:	[KSTREAM-SINK-0000000014]
		KSTREAM-SINK-0000000014:
			topic:		tec1.trex-jcheng.trex-delta.pgd-deltas-series
> Partitions [tec1.trex-jcheng.trex-delta.replicatorFullExportPartitioned-0]
```

Reply:
Cool, that's helpful. Thanks.